spot_driver/src/images/spot_image_publisher_node.cpp

```diff
@@ -39,7 +39,8 @@ SpotImagePublisherNode::SpotImagePublisherNode(const rclcpp::NodeOptions& node_o
   auto tf_broadcaster = std::make_unique<RclcppTfBroadcasterInterface>(node);
   auto timer = std::make_unique<RclcppWallTimerInterface>(node);
 
-  auto spot_api = std::make_unique<DefaultSpotApi>(kSDKClientName, parameters->getCertificate());
+  auto timesync_timeout = parameters->getTimeSyncTimeout();
```

Comment:
nit: this (and other calls to getTimeSyncTimeout in other nodes) can be const 
```suggestion
  const auto timesync_timeout = parameters->getTimeSyncTimeout();
```